src/sql/azdata.proposed.d.ts

```diff
@@ -680,13 +706,29 @@ declare module 'azdata' {
 
 		export type DialogWidth = 'narrow' | 'medium' | 'wide' | number;
 
+		export type DialogStyle = 'normal' | 'flyout' | 'callout' | 'calloutCompact';
+
+		export type DialogPosition = 'left' | 'below';
+
+		export interface IDialogProperties {
+			xPos: number,
+			yPos: number,
+			width: number,
```

Comment:
1. why do we have a IDialogProperties interface only including these 4 properties and not including other dialog properties? 
2. there is already a dialogWidth property, we shouldn't introduce a new width property.

Reply:
This is a new interface. I will add comments so that it is understood.